akkeeper/src/main/scala/akkeeper/common/Controllers.scala

```diff
@@ -13,23 +13,27 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package akkeeper.master.route
+package akkeeper.common
 
 import akka.actor.ActorRef
-import akka.pattern.ask
 import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport
 import akka.http.scaladsl.model.StatusCode
-import akka.http.scaladsl.server.{Directives, Route}
+import akka.http.scaladsl.server.{Directives, PathMatchers, Route}
+import akka.pattern.ask
 import akka.util.Timeout
 import akkeeper.AkkeeperException
 import akkeeper.api.CommonApiJsonProtocol
 import spray.json.RootJsonWriter
+
 import scala.collection.mutable
 import scala.reflect.ClassTag
 
-trait BaseController extends Directives with SprayJsonSupport with CommonApiJsonProtocol {
-
+trait RouteController extends Directives with SprayJsonSupport {
```

Comment:
`Route` prefix doesn't really add any information to the `Controller` itself. Can we perhaps just name it `Controller`?